### ISSUE TITLE
Add watershed boundary layers (like StreamStats) 

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/FEV#214]
Stick with the level of detail that StreamStats offers - which changes depending on zoom level.

See grey lines in the image:
![image](https://user-images.githubusercontent.com/12058013/63718414-835a1500-c818-11e9-9dce-9c02bf65b033.png)

**Comment by lprivette** [comment]
@HansVraga that's the 'Dark Grey' esri basemap right? So for this issue we just want to add this basemap as an option?

**Comment by HansVraga** [comment]
no - the lines are a webservice separate from the Dark Grey basemap.

**Comment by lprivette** [comment]
Ok, this is currently being done in StreamStats using these layers https://gis.streamstats.usgs.gov/arcgis/rest/services/StreamStats/nationalLayer/MapServer/1

sub layers == the hucs

I'm not sure how they have it setup in their code but here is some more history on it https://github.com/USGS-WiM/StreamStats/issues/705

**Comment by 09Kathy** [comment]
when I try to load these layers (or just one huc level), it says there's no data. All the rest services I've used before have a little data map preview, and this one doesn't, which maybe suggests there's no longer data tied to it. The link in the SS issue doesn't have data in there either:
https://www.arcgis.com/home/webmap/viewer.html?url=https://services.nationalmap.gov/arcgis/rest/services/wbd/MapServer&source=sd
I downloaded the SS project and couldn't find where it was loaded there either (except there was a path for all national layers so maybe they did it without getting the watersheds explicitly). 
I definitely could be trying to upload it incorrectly, but I want to see if anyone knows that this data source is current/valid before I go down a rabbit hole.
